distribution.pm

```diff
@@ -118,26 +118,29 @@ to return.
 =cut
 
 sub script_run {
-    # start console application
-    my ($self, $cmd, $wait) = @_;
-    $wait //= $bmwqemu::default_timeout;
+    my ($self, $cmd) = splice(@_, 0, 2);
+    my %args = OpenQA::Isotovideo::Utils::compat_args(
+        {
+            'timeout' => $bmwqemu::default_timeout,
```

Comment:
Perlcritic likely don't like the "unnecessary" quotes around the keys.

Reply:
ups left over... thx